Vagrantfile

```diff
@@ -28,6 +28,8 @@ Vagrant.configure('2') do |config|
 
   # Forwarded port mapping for http
   config.vm.network 'forwarded_port', guest: 3000, host: 3000
+  # Forwarded port mapping for rest-api
+  config.vm.network 'forwarded_port', guest: 8080, host: 8080
```

Comment:
While you're updating the Vagrant file, what's required for redis to work within Vagrant?